Tests/StackClusterTests.swift

```diff
@@ -161,7 +163,7 @@ class StackClusterTests: QuickSpec {
                     it("should be able to scroll horizontally") {
                         let vc = nestedStack()
                         vc?.loadViewIfNeeded()
-                        
+                        vc?.view.layoutIfNeeded()
```

Comment:
@joanromano I added it to every testcase, that would fail, if the check with `haveValidSnapshot` was not called. Checking for a snapshot forces the layout, but if you remove the check and then just check for `expect(scrollView).to(scroll(.horizontal))` the test would fail.